tests/phpunit/test-class-wc-stripe-upe-payment-gateway.php

```diff
@@ -156,6 +156,83 @@ private function get_order_details( $order ) {
 		return [ $amount, $description, $metadata ];
 	}
 
+	/**
+	 * @dataProvider get_upe_available_payment_methods_provider
+	 */
+	public function test_get_upe_available_payment_methods( $country, $available_payment_methods ) {
+		$this->set_stripe_account_data( [ 'country' => $country ] );
+		$this->assertSame( $available_payment_methods, $this->mock_gateway->get_upe_available_payment_methods() );
+	}
+
+	/**
+	 * @dataProvider get_upe_enabled_at_checkout_payment_method_ids_provider
+	 */
+	public function test_get_upe_enabled_at_checkout_payment_method_ids( $country, $available_payment_methods ) {
```

Comment:
Props for adding these tests.

If you agree with restricting Link only to the `get_upe_available_payment_methods` method, you can remove this other test.

Reply:
changed in [0ea817f](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2452/commits/0ea817f020274f678061c27bbd199a874a2578e8) .  thx